### ISSUE TITLE
Having additional checks in case `./apis/v1alpha1` is missing.

### ISSUE THREAD

**Issue by remyleone** [crossplane/terrajet#222]
### What happened?

When I was trying to debug code generation I didn't knew that `./apis/v1alpha1` should be kept. I've deleted it and it was not evident for me to go back to the template to copy and paste this boilerplate.

Maybe it could be a good idea to have additional checks in angryjet to detect if this folder/file is missing. It could even be generated from angryjet.
If it cannot, having a README.md in the api folder of the template could be helpful to understand the folder that should be kept and the one that will be generated.

**Comment by turkenh** [comment]
At some point we also had some target like `make terrajet.clean` which removes all generated files to have a fresh start. We removed that after adding this cleanup as an [initial step](https://github.com/crossplane-contrib/provider-jet-template/blob/d656e1e9be8101434ffbd6b94473518c78cf4337/apis/generate.go#L26-L29) for `make generate` as it would not be required to be called explicitly. 

Maybe we can add something similar just to better point out what is generated or not and have a way to start fresh :)

**Comment by remyleone** [comment]
Yes, it would be very helpful to have a command that cleans and generate everything again from scratch. It would lower the mental load to think about what are the generated files and what are the files that are not. It was not easy for me to understand what files should remain and what files should be completed by hand.
